profiles/orcid.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+import requests
+from requests import Response
+
+API_VERSION = 'v2.0'
+
+
+class OrcidClient(object):
```

Comment:
Doesn't look to be a usable Python library for the 2.0 API. https://github.com/ORCID/python-orcid looks unstable.

Reply:
Agreed, you don't want to be cut-off from a functionality because a library is not up-to-date.